config/locales/en.yml

```diff
@@ -58,6 +58,18 @@ en:
     update:
       success: "Confirmation updated."
 
+  cyclists:
+    new:
+      header: "Invite a Cyclist"
+    show:
+      header: "Cyclist: %{name}"
+      instructions:
+        >
+        A cyclist account has been created for %{email}.
+
+        Instruct them to visit the following link to reset their password
+        and gain access to the latest pickup checklist.
```

Comment:
I think the header is sufficient. I might say "Please instruct the cyclist" instead of "Instruct them"